jquery.marquee.js

```diff
@@ -143,7 +143,10 @@
             });
 
             if (o.duplicated) {
-                $el.clone(true).appendTo($this);
+                var max = o.duplicated > 1 ? o.duplicated : 1;
```

Comment:
this will covert duplicated from a boolean to a number.
Can you update the Readme to replicate that, please?